apps/examples/fs_helper/fs_rw_test.c

```diff
@@ -0,0 +1,168 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define TEST_FILE_PATH "/mnt/"
```

Comment:
In my opinion, you had better get path from user input.
If you fix root path as a '/mnt' , it means this example can test only smartfs.
To test tmpfs and other driver's performance, you'd better accept my suggestion.

Reply:
Give me your answer.

Reply:
modify get path from user input.